auth-service/src/main/kotlin/uk/gov/communities/delta/auth/plugins/StatusPages.kt

```diff
@@ -7,11 +7,20 @@ import io.ktor.server.plugins.statuspages.*
 import io.ktor.server.request.*
 import io.ktor.server.response.*
 import io.ktor.server.thymeleaf.*
+import uk.gov.communities.delta.auth.config.DeltaConfig
 
 private const val apiRoutePrefix = "/auth-internal/"
 
 fun Application.configureStatusPages(deltaWebsiteUrl: String) {
     install(StatusPages) {
+        // Currently only the login page is rate limited so always redirects to the login page for status "TooManyRequests"
```

Comment:
```suggestion
        // Currently only the login page is rate limited so always renders to the login page for status "TooManyRequests"
```
We're responding directly to the request, not redirecting